bin/aq.py

```diff
@@ -347,8 +347,8 @@ def get_default_opts(auth_option, conf_file=None):
         default_aqhost = defaultOpts.get('aqhost') or 'nyaqd1'
         default_aqservice = 'cdb'
     else:
-        default_aqhost = socket.gethostname()
-        default_aqservice = get_username()
+        default_aqhost =  defaultOpts.get('aqhost') or socket.gethostname()
+        default_aqservice =  defaultOpts.get('aqservice') or get_username()
 
     host = globalOptions.get('aqhost') or os.environ.get('AQHOST', None) or \
```

Comment:
Actually aqservice is already not ignoring aq.conf default aqservice value:
https://github.com/jouvin/aquilon/blob/3a416e34347731196868be30156866828f78de33/bin/aq.py#L377

I suggest to leave this line alone and just add additional 'or defaultOpts.get('aqservice')'  after globalOptions.get('aqhost') in the line #353 for host.